src/cli/simple-orchestrator.ts

```
@@ -700,9 +700,9 @@ function startWebUI(host: string, port: number) {
     const cmd = args[0];
     const cmdArgs = args.slice(1);
 
-    // Determine the correct claude-flow executable path
+    // Determine the correct auggie-flow executable path
     const rootDir = path.resolve(__dirname, '../..');
-    const cliPath = path.join(rootDir, 'bin', 'claude-flow');
+    const cliPath = path.join(rootDir, 'bin', 'auggie-flow');
```

ESLint

[error] 'cliPath' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)